src/Native/CMakeLists.txt

```diff
@@ -127,6 +127,13 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
     set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,--build-id=sha1")
     set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -Wl,--build-id=sha1")
     add_compile_options(-fstack-protector-strong)
+    
+    # Support 16 KB page sizes for Android
+    if(ANDROID)
```

Comment:
We don't actually have a separate android build, so this does nothing.  I checked and dotnet/runtime only sets this for it's android builds, so they had some reason not to do this for all linux platforms.

We could try to set it for all linux platforms, but it might have perf side-effects elsewhere.  I'm also not inclined to add an android specific build to the package unless there is a strong demand for it.